SAS/ttest.qmd

```diff
@@ -0,0 +1,78 @@
+---
+title: "Students t-test"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+### **Independent t-test in SAS**
+
+The null hypothesis of the Independent Samples t Test is, the means for the two populations are equal.
+
+In SAS the following code was used to test the mean comparison (mean of Weight Gain) of two independent treatment groups (Treatment and Placebo).
+
+For this example, we're testing the significant difference in mean of Weight gain (*WtGain*) between treatment and placebo (*trt_grp*) using PROC TTEST procedure in SAS.
+
+```{r}
+#| eval: false 
+  proc ttest data=d1; 
+     class trt_grp; 
+     var WtGain; 
+  run; 
+```
+
+Output:
+
+```         
+             Figure 1: Test results for independent t-test using PROC TTEST in SAS
+```
+
+```{r, echo=FALSE, fig.align='center', out.width="50%"}
+knitr::include_graphics("../images/ttest/test1.png")
+```
+
+Here the t value is --0.70, degrees of freedom is 30 and P value is 0.4912 which is greater than 0.05, so we accept the null hypothesis that there is a significant difference between the means of treatment groups. Also the mean difference is --7.9375 and the 95% CI for the mean difference is [--31.1984, 15.3234].
+
+Note: Before entering straight into the t-test we need to check whether the assumptions (like the equality of variance, the observations should be independent, observations should be normally distributed) are met or not. If normality is not satisfied, we need to go with the non-parametric test of independent t test.
+
+1.  Normality: We have checked the normality of the observations using the code below. Here for both the treatment groups we have P value greater than 0.05 (Shapiro-Wilk test is used), therefore the normality assumption is there for our data.
+
+```{r}
+#| eval: false 
+    proc univariate data=d1 normal;  
+      qqplot WtGain; 
+      by trt_grp; 
+    run; 
+```
+
+Output:
+
+```         
+        Figure 2: The results of normality test for Treatment group
+```
+
+```{r, echo=FALSE, fig.align='center', out.width="30%"}
+knitr::include_graphics("../images/ttest/trt_sas.png")
+```
+
+```         
+       Figure 3: The results of normality test for Placebo group
+```
+
+```{r, echo=FALSE, fig.align='center', out.width="30%"}
+knitr::include_graphics("../images/ttest/placb_sas.png")
+```
+
+2.  Homogeneity of variance (or Equality of variance): Homogeniety of variance will be tested by default in PROC TTEST itself by Folded F-test. In our case the P values is 0.6981 which is greater than 0.05. So we accept the null hypothesis of F test, i.e. variances are same. Then we will consider the pooled method for t test.
+
```

Comment:
Suggest to add after "we will consider the pooled method for t test".
If the F test is statistically significant (p<0.05), then the pooled t-test may give erroneous results.  In this instance, if it is believed that the population variances may truly differ, then the Satterthwaite (unequal variances) analysis results should be used.  These are provided in the SAS output alongside the Pooled results as default.